crc/aarch64/crc_aarch64_dispatcher.c

```diff
@@ -81,6 +87,8 @@ DEFINE_INTERFACE_DISPATCHER(crc32_iscsi)
 	if (auxval & HWCAP_PMULL) {
 		return PROVIDER_INFO(crc32_iscsi_refl_pmull);
 	}
+#elif defined(__aarch64__)
+	return PROVIDER_INFO(crc32_iscsi_refl_pmull);
```

Comment:
this function might not be best choice . As I know, crc32_iscsi_crc_ext or crc32_iscsi_3crc_fold should be better choice.

Reply:
Well, I'm sorry.

Although I have asked an acquaintance of mine to test the **same** binary on ARM mac (it worked, so "it does support ARM mac"), my primary machine is Intel mac and my test environment is iPad (well, jailbroken and sshd enabled).

Also, at least crc32 instruction caused SIGILL on my iPad (from libslz).

The worse thing, as far as I know, there are no `runtime cpu feature detection API` on Darwin. That's why I adjust the dispatcher to middle-range...

Reply:
It seems undocumented `_get_cpu_capabilities` can be used as "runtime cpu feature detection API".

Now my concern is https://developer.apple.com/documentation/xcode/writing_arm64_code_for_apple_platforms `The platforms reserve register x18. Don’t use this register.` Allowing CRC32 instruction could get into this codepath...

Reply:
`x18` problem should be fix . I will raise an issue later.

And it looks `runtime cpu feature detection ` should be added in Apple. As I known , M1 (mac mini arm64 ) are available now. I guess it has more feature support. 

Could you review `aarch64_multibinary.h` ? I am not sure if there are anything that can not match Apple spec.